docs/user_guide/web-components.rst

```diff
@@ -132,14 +132,17 @@ Here are some of the available button-style links, also using semantic colors:
 .. note::
 
    `Sphinx Design buttons
-   <https://sphinx-design.readthedocs.io/en/latest/badges_buttons.html>`__
-   are actually links, meaning they are rendered in HTML with ``<a>`` tags
-   instead of ``<button>``. Use them if you need a link to look like a button,
-   however, be aware that they do not follow accessibility best practices for
-   native button components such as using the correct `ARIA attributes
-   <https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role>`__.
```

Comment:
I felt that this did not help the theme adopter because it did not make a recommendation nor was the info helpful enough for a non-accessibility expert to make a judgment call, so I updated it with a specific recommendation (to use sparingly) and a specific example (about how the space bar doesn't work on links that look like buttons).